templates/fonts-default.stencil

```diff
@@ -0,0 +1,29 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+{% if families %}
+import UIKit.UIFont
+
+extension UIFont {
+{% for family in families %}
+    convinience init?(font: {{enumName}}.{{family.name|swiftIdentifier|snakeToCamelCaseNoPrefix}}, size: CGFloat) {
```

Comment:
`convinience` -> `convenience`

Also I think you might instead leverage `protocol` with default implementations (aka [Mixins](http://alisoftware.github.io/swift/protocol/2015/11/08/mixins-over-inheritance/)) here.
You can get some inspiration from what I did in the `storyboards-default.stencil` template. This would probably give something like this:

```swift
protocol FontConvertible {
  func font(size: CGFloat) -> UIFont!
}

extension FontConvertible where Self: RawRepresentable, Self.RawValue == String {
  func font(size: CGFLoat) -> UIFont! {
    return UIFont(name: self.rawValue, size: size)
  }
}

extension UIFont {
  convenience init!(font: FontConvertible, size: CGFloat) {
    let font = font.font(size)
    self.init(name: font.fontName, size: size)
  }
}
```

This way, you can avoid that loop to create as many convenience init as font families in `UIFont`, and also avoid redeclaring the `func font` for each enum: simply mark them to be conform to `FontConvertible` and they will get that implementation for free.

---

PS: Given that the enums are auto-generated by SwiftGen, I think that returning a `UIFont!` here instead of an `UIFont?` is OK there, as if the font can't be created from the enum, that can only be a developer error — probably because the developer forgot to re-run SwiftGen before compiling.